src/main/java/com/mitchej123/hodgepodge/mixins/late/extrautilities/MixinBlockSpike.java

```diff
@@ -0,0 +1,32 @@
+package com.mitchej123.hodgepodge.mixins.late.extrautilities;
+
+import com.mitchej123.hodgepodge.Hodgepodge;
+import com.rwtema.extrautils.block.BlockSpike;
+import net.minecraft.entity.Entity;
+import net.minecraft.world.World;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(BlockSpike.class)
```

Comment:
I think this can accept multiple classes in an array, which would mean we only need one mixin as opposed to 4.

Reply:
I went ahead and did this, going to merge.  Would be nice if you could verify my change works @TimeConqueror